### ISSUE TITLE
subtitles

### ISSUE THREAD

**Issue by buzzbo** [mmitch/omxplayer-frontend#3]
Is there a way to turn off subtitles while video is playing? I just happened to notice this because I have a matching srt file in the directory of a video. Looks like it can be toggled.

**Comment by mmitch** [comment]
When omxplayer has a key for that, you could add a new command to omxplayer.
Look for 'languageup' and 'languagedown' for what to do: both commands map to a key and display as a button in the web page.
I don't use omxplayer-frontend any more, so propably I won't update it any more.
I've switched to omxplayer-console which is able to disable subtitles (but it has no web frontend, it's for console use).

**Comment by buzzbo** [comment]
OK a couple things. 'languageup' and 'languagedown' were listed as j and k which conflict with key commands in omxplayer 'nextaudio' 'prevaudio':

    command_send={
    'speedup':'1',
    'speeddown':'2',
    'nextaudio':'k',
    'prevaudio':'j',
    'nextchapter':'o',
    'prevchapter':'i',
    'nextsubs':'m',
    'prevsubs':'n',
    'togglesubs':'s',
    'stop':'q',
    'pause':'p',
    'volumedown':'-',
    'volumeup':'+',
    'languagedown':'j',
    'languageup':'k',
    'seek-30':'\x1b\x5b\x44',
    'seek+30':'\x1b\x5b\x43',
    'seek-600':'\x1b\x5b\x42',
    'seek+600':'\x1b\x5b\x41'}

What I did was map them to n 'previous subtitle stream' and m 'next subtitle stream' which seems like your languageup and languagedown. I'm not really sure but it did not toggle it.

Then I noticed togglesubs 's' so I added a button and edited the javascript and html in interface.htm, mapping 't' to togglesubs, and was able to successfully toggle the subtitles. Not sure what glyph to use for the toggle as CC seems off center and cc is lowercase.

I'm not really a dev and I use github as a user. I noticed the code is at bitbucket. Here are my changes: http://pastebin.com/ShLteCg4
